cloud/scope/azurestackedgeippool.go

```diff
@@ -146,60 +146,60 @@ func (s *AzureIPPoolScope) ReconcileIPs(ctx context.Context, machineScope *Machi
 				return errors.Wrapf(err, "Error in getting next free IP for the machine %s",machineScope.Name())
 			}
 			networkInterface.StaticIPAddress = allocatedIP
```

Comment:
Should we catch IP address in use, by doing ARP request on ASE? not sure whether it should be added in NRP or part of MSK8 workflow

Reply:
I am not sure if we can execute the command against ASE from CAPE. Can we do it in our kube service in ASE code? Check all the IPs that are a part of IP pool if they are free or not?

Reply:
you can discuss with Jia, whether this can be added in NRP otherwise, we should add it in kube service in ASE code.